client/src/Job.jsx

```diff
@@ -162,7 +162,7 @@ class Job extends React.Component{
             statusText.push('loading...');
         }
         var link = clickable
-            ? <a onClick={function () { this.handleClick(clickWarning); }.bind(this)} href="#">{this.props.children}</a>
+            ? <button className="btn-link" onClick={function () { this.handleClick(clickWarning); }.bind(this)}>{this.props.children}</button>
```

Comment:
Does changing this break back button behavior? I toyed with changing this, but it seemed to break the ability to go back from the job editing screen to the job selection screen, because without the href="#", clicking into a job doesn't add anything to browser history. I think it would be better to solve this in the future with react-router (which might enable us to make the whole application more elegant), but in the meantime I want to be sure we don't lose functionality.

Reply:
I think proper routing is the way to go for sure. I can remove this change or we can just refresh the page to get back to the jo picker.

Reply:
Maybe exploring react-router can be a future group meeting project (though it's definitely not top priority right now -- I want to get functionality in place first, but we should probably explore routing before we start porting the Orbeon functionality).

For now, what if we update the Javascript callback to add a hash to the browser history? Would that maintain the existing functionality while also suppressing the warning? We could also put in a TODO comment to remind ourselves to replace it with real routing when we get that far.

Reply:
I switched it back to a link with a fancier hash to make the warning happy but going back isn't returning to the job list.

Would it be worth adding an explicit back to selector link?

Reply:
Either way, I reverted it to a link in case it's just my box.

Reply:
Weird, I'll have to test this when I'm back in the office.

Reply:
It might be worth adding an explicit back control to the UI, but it would definitely be nice to keep the existing back button behavior working, since I think everyone is accustomed to it by now.